test/semantics/resolve45.f90

```diff
@@ -1,76 +0,0 @@
-! Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
This looks like a merge glitch. This file shouldn't be removed.

Reply:
Right.